server/telegram.go

```diff
@@ -86,6 +92,8 @@ func (t *telegramHandler) MessageHandle(ctx context.Context, bot *telegram.Bot,
 			mime = "image/jpeg"
 		case ".png":
 			mime = "image/png"
+		case ".pdf":
```

Comment:
Could you please add more mime support, such as videos?

Reply:
Here you need to look at the situation. So far I have added mp4. In the other cases that I tested, mime is taken from the data in the message